common/lib/capa/capa/util.py

```diff
@@ -65,24 +65,18 @@ def compare_with_tolerance(student_complex, instructor_complex, tolerance=defaul
         # `tolerance` both equal to infinity. Then, below we would have
         # `inf <= inf` which is a fail. Instead, compare directly.
         return student_complex == instructor_complex
+
+    if isinstance(student_complex, float) and isinstance(instructor_complex, float):
```

Comment:
Interesting--so we really shouldn't be using floats in `test_util.py`. Could we change the test then to:
```
if student_complex.imag == 0 and instructor_complex.imag == 0:
    student_decimal = Decimal(str(student_complex.real))
```
etc?